src/components/Tabs.js

```diff
@@ -0,0 +1,203 @@
+const React = require('react');
+const Radium = require('radium');
+
+const Icon = require('./Icon');
+const SimpleSelect = require('./SimpleSelect');
+const StyleConstants = require('../constants/Style');
+
+const Tabs = React.createClass({
+  propTypes: {
+    brandColor: React.PropTypes.string,
+    onTabSelect: React.PropTypes.func.isRequired,
+    selectedBottomBorderPadding: React.PropTypes.number,
+    selectedTab: React.PropTypes.number,
+    showBottomBorder: React.PropTypes.bool,
+    tabs: React.PropTypes.array.isRequired
+  },
+
+  getDefaultProps () {
+    return {
+      brandColor: StyleConstants.Colors.PRIMARY,
+      showBottomBorder: true,
+      selectedBottomBorderPadding: 5
+    };
+  },
+
+  getInitialState () {
+    return {
+      selectedTab: this.props.selectedTab || 0,
+      showMenu: false
+    };
+  },
+
+  componentWillReceiveProps (nextProps) {
+    if (nextProps.selectedTab !== this.state.selectedTab) {
+      this.setState({
+        selectedTab: nextProps.selectedTab
+      });
+    }
+  },
+
+  _handleMenuClick () {
+    this.setState({
+      showMenu: !this.state.showMenu
+    });
+  },
+
+  _handleTabClick (selectedTab) {
+    this.props.onTabSelect(selectedTab);
+    this._handleMenuClick();
+
+    this.setState({
+      selectedTab
+    });
+  },
+
+  _isLargeOrMediumWindowSize () {
+    const windowSize = StyleConstants.getWindowSize();
+
+    return windowSize === 'medium' || windowSize === 'large';
+  },
+
+  _renderTabs () {
+    const selectedTabStyle = {
+      borderBottom: '2px solid ' + this.props.brandColor,
+      paddingBottom: this.props.selectedBottomBorderPadding
+    };
+    const styles = this.styles();
+
```

Comment:
I think we should allow a style props and object assign with the defined function.